src/core/client/ui/components/icons/FlairNotFoundIcon.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FunctionComponent } from "react";
+
+const FlairNotFoundIcon: FunctionComponent = () => {
```

Comment:
For consistency with the other icons' naming conventions, and so that the name is more flexible for reuse, can you please rename to `ImageFileWarningIcon`?

Can you also export this icon from `index.ts` like the others for consistency and so we can just import all from `coral-ui/components/icons`?

Reply:
Fixed!